test/end-to-end/utils/dbAssertions.js

```diff
@@ -50,32 +50,31 @@ const checkThatHealthAssessmentDataWasWrittenToDatabase = ({
       return result[0];
     });
 
-const checkThatPrisonerAssessmentDataWasWrittenToDatabase = ({
+export function checkThatPrisonerAssessmentDataWasWrittenToDatabase({
   id,
   nomisId,
   forename,
   surname,
   dateOfBirth,
-}) => db.select()
-    .table('prisoner_assessments')
-    .where('id', Number(id))
-    .then((result) => {
-      expect(result[0]).to.not.equal(
+}) {
+  return db.select().table('prisoner_assessments').where('id', Number(id)).then((result) => {
```

Comment:
Curious whether you could rewrite this method using async await by removing the then part?